src/lib/SapphireClient.ts

```diff
@@ -19,10 +19,109 @@ import type { Awaited } from './utils/Types';
 // Extensions
 import './extensions/SapphireMessage';
 
+/**
+ * A valid prefix in Sapphire.
+ * * `string`: a single prefix, e.g. `'!'`.
+ * * `string[]`: an array of prefixes, e.g. `['!', '.']`.
+ * * `null`: disabled prefix, locks the bot's command usage to mentions only.
+ */
+export type SapphirePrefix = string | readonly string[] | null;
+
 export interface SapphirePrefixHook {
-	(message: Message): Awaited<string | readonly string[] | null>;
+	(message: Message): Awaited<SapphirePrefix>;
+}
+
+export interface SapphireClientOptions {
+	/**
+	 * The base user directory, if set to `null`, Sapphire will not call [[SapphireClient.registerUserDirectories]],
+	 * meaning that you will need to manually set each folder for each store. Please read the aforementioned method's
+	 * documentation for more information.
+	 * @since 1.0.0
+	 * @default undefined
+	 */
+	baseUserDirectory?: string | null;
+
+	/**
+	 * The default prefix, if it returns `null`, only mention prefix will trigger the bot's commands.
```

Comment:
```suggestion
	 * The default prefix, in case of `null`, only mention prefix will trigger the bot's commands.
```

`SapphirePrefix` is not a function so it doesn't "return"